website/src/content/deployment/compile-html.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Compiling Custom Application
```

Comment:
Should we name this file `compiling-index-html.mdx`? And the title `Compiling index.html`?

Just looking at this from the navigation menu links perspective...

Reply:
Technically it's only about compiling the `index.html` file. Maybe something like `Compiling the application for static hosting`, or `Preparing the application for static hosting`?

We can also get some input from others (cc @tdeekens @anilkk )

PS: The file name should more or less match the title.

Reply:
I don't have strong opinions here apart from title matching filename. Given we ever compile more html than the `index.html` keeping it "generic" could help. As a consumer I also don't care that the thing I compile is an `index.html` it's more about the fact that I have to. Maybe "Compile Custom Application index file"